sources/api/schnauzer/src/helpers.rs

```diff
@@ -907,6 +910,128 @@ pub fn goarch(
     Ok(())
 }
 
+handlebars_helper!(is_ipv4: |cidr: str| {
```

Comment:
Nice! It would be awesome to also add an `is_ipv6` for the API, but not a blocker.